src/ol_openedx_course_export/utils.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+
+from ol_openedx_course_export.constants import AWS_S3_DEFAULT_URL_PREFIX
+
+
+def is_bucket_configuration_valid():
+    """
+    For course export to work properly we need all the AWS settings configured properly
+    """
+    return None not in [
+        settings.COURSE_IMPORT_EXPORT_BUCKET,
+        settings.AWS_ACCESS_KEY_ID,
+        settings.AWS_SECRET_ACCESS_KEY,
```

Comment:
We should allow for empty AWS keys so that we can use IAM instance role permissions on the EC2 machines.

Reply:
I will remove these from the configuration checks except `settings.COURSE_IMPORT_EXPORT_BUCKET`. moreover, even if there would be an error in access it would be logged within the API exception catch.

Reply:
This is incorporated.